ios/FluentUI/Tab Bar/SideTabBar.swift

```diff
@@ -322,6 +322,15 @@ open class SideTabBar: UIView {
         return nil
     }
 
+    /// This method returns an optional view for the given tab bar item.
```

Comment:
nit: remove "This method" and make it just:

"Returns the first match of an optional view for a given tab bar item."